config/config.go

```diff
@@ -179,11 +180,12 @@ type OfflineModeConfig struct {
 // variables, individual fields are not documented here; instead, see the `README.md` section on
 // configuration.
 type EventsConfig struct {
-	EventsURI     ct.OptURLAbsolute        `conf:"EVENTS_HOST"`
-	SendEvents    bool                     `conf:"USE_EVENTS"`
-	FlushInterval ct.OptDuration           `conf:"EVENTS_FLUSH_INTERVAL"`
-	Capacity      ct.OptIntGreaterThanZero `conf:"EVENTS_CAPACITY"`
-	InlineUsers   bool                     `conf:"EVENTS_INLINE_USERS"`
+	EventsURI                ct.OptURLAbsolute        `conf:"EVENTS_HOST"`
+	SendEvents               bool                     `conf:"USE_EVENTS"`
+	FlushInterval            ct.OptDuration           `conf:"EVENTS_FLUSH_INTERVAL"`
+	Capacity                 ct.OptIntGreaterThanZero `conf:"EVENTS_CAPACITY"`
+	InlineUsers              bool                     `conf:"EVENTS_INLINE_USERS"`
+	MaxReceivablePayloadSize units.Base2Bytes         `conf:"EVENTS_MAX_RECEIVABLE_PAYLOAD_SIZE"`
```

Comment:
I would welcome a better name.

Reply:
How about: `MaxInboundPayloadSize` and `EVENTS_MAX_INBOUND_PAYLOAD_SIZE`?


If we added a config for the max that could be sent to LD, we could call it `Outbound`.

Reply:
Updated to this new name.